### ISSUE TITLE
Error With ADB occured[BUG]

### ISSUE THREAD

**Issue by Muhd-Mubi** [Cyrexxis/RokTracker#88]
![image](https://github.com/Cyrexxis/RokTracker/assets/49670609/7003c495-25bf-4538-b32d-3b0b3ea814f1)
when done `adb devices` via powershell it shows that emulator-'''''' status offline
adb bridge via bluestack is open
![image](https://github.com/Cyrexxis/RokTracker/assets/49670609/e4a3f960-fed0-43bc-934d-a98811291621)
![image](https://github.com/Cyrexxis/RokTracker/assets/49670609/515c5c53-dfa3-4f3d-bf44-8dda641a8a27)
I tried reinstalling bluestack, reinstalling adb drivers and platform tools
![image](https://github.com/Cyrexxis/RokTracker/assets/49670609/b6452ed3-2ebf-4631-8d12-0213475ba59d)
maybe an error with adb or bluestack because other devices connect properly.

**Comment by Muhd-Mubi** [comment]
> ![image](https://private-user-images.githubusercontent.com/49670609/317248805-7003c495-25bf-4538-b32d-3b0b3ea814f1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE1NDAyMTgsIm5iZiI6MTcxMTUzOTkxOCwicGF0aCI6Ii80OTY3MDYwOS8zMTcyNDg4MDUtNzAwM2M0OTUtMjViZi00NTM4LWIzMmQtM2IwYjNlYTgxNGYxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI3VDExNDUxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcxNmJhODhkMDJkZjg2YjhlMWQ1NTMwZTU3Zjg5MGFkOWE1YWJiNDViYjNkOTE4MmRmMGJjNDZjZDJlZWM4ODYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Q0ONkAihezu5b85e9i4OJn5ZcSxwwXa_-6DzmpENftE) when done `adb devices` via powershell it shows that emulator-'''''' status offline adb bridge via bluestack is open ![image](https://private-user-images.githubusercontent.com/49670609/317249166-e4a3f960-fed0-43bc-934d-a98811291621.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE1NDAyMTgsIm5iZiI6MTcxMTUzOTkxOCwicGF0aCI6Ii80OTY3MDYwOS8zMTcyNDkxNjYtZTRhM2Y5NjAtZmVkMC00M2JjLTkzNGQtYTk4ODExMjkxNjIxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI3VDExNDUxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVmNzYxNGE4ZDk1MWU4ZjgyNGE4ZGNlYTYzYzIyYmM0YWJiODJkMWY4ZjNjMjBlNzYyN2I3NDIyYmFmZDEyN2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.U11q4RH7ETwzSP5aZsa0VfXaFXH_1P4e5iPEWfBCufE) ![image](https://private-user-images.githubusercontent.com/49670609/317249463-515c5c53-dfa3-4f3d-bf44-8dda641a8a27.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE1NDAyMTgsIm5iZiI6MTcxMTUzOTkxOCwicGF0aCI6Ii80OTY3MDYwOS8zMTcyNDk0NjMtNTE1YzVjNTMtZGZhMy00ZjNkLWJmNDQtOGRkYTY0MWE4YTI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI3VDExNDUxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxODcyZDRhNGI2M2Q4NDhhNTc4ZmU5OTc1YzQzMzM0NWY0N2YwNzg5NjY1N2Q4OWZmZmJkMTE3Y2Q3YTc4YTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.xecbZ7fdWGPu5HNEU8vEnJw39-pCvhGzRJ3gCCeO-uw) I tried reinstalling bluestack, reinstalling adb drivers and platform tools ![image](https://private-user-images.githubusercontent.com/49670609/317249848-b6452ed3-2ebf-4631-8d12-0213475ba59d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE1NDAyMTgsIm5iZiI6MTcxMTUzOTkxOCwicGF0aCI6Ii80OTY3MDYwOS8zMTcyNDk4NDgtYjY0NTJlZDMtMmViZi00NjMxLThkMTItMDIxMzQ3NWJhNTlkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI3VDExNDUxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBjNjE2OTQ2Y2ZiMjA3OTc4NDM2M2JlYTViMjJiYmQzNzU1OWMzZjYzOGUwYzRlMWRlZTUxZjdjNjYwZWJjM2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4ZK7XaW0Rm7kkhpTTVhQD4eV-a6cyQsDm8yYNWZFaks) maybe an error with adb or bluestack because other devices connect properly.

yes I did manually put port 5556 in correctly multiple times, forgot to mention that

**Comment by Cyrexxis** [comment]
Hm, that is a very strange case. Are you sure the scanner tries to use the correct port? On the other hand, normally bluestacks also should show up in the normal adb... One thing you can try with the normal adb is ```adb connect 127.0.0.1:5556``` (5556 is the port from your screenshot, replace with your bluestacks port if it doesn't match anymore).

If that works, let me know what the log of the scanner is when you use the same port there.
